app/src/main/java/fr/paug/androidmakers/ui/fragment/VenueFloorPlanFragment.kt

```diff
@@ -0,0 +1,36 @@
+package fr.paug.androidmakers.ui.fragment
+
+import android.graphics.BitmapFactory
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.ScrollView
+import androidx.appcompat.widget.AppCompatImageView
+import androidx.fragment.app.Fragment
+import fr.paug.androidmakers.R
+
+class VenueFloorPlanFragment: Fragment() {
+    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+        val scrollView = ScrollView(context!!)
```

Comment:
I would wrap everything inside: 
```kotlin
context?.let {
}
```
instead of using `!!`, in case it's not attached to the activity